aopy/analysis/base.py

```diff
@@ -2231,4 +2231,53 @@ def windowed_xval_lda_wrapper(data, labels, samplerate, lags=3, nfolds=5, regula
     elif return_confusion_matrix:
         return decoding_accuracy, time_axis, cm
     else:
-        return decoding_accuracy, time_axis
\ No newline at end of file
+        return decoding_accuracy, time_axis
+    
+def simulate_trajectories(targets, origin=[0.0, 0.0, 0.0], resolution=100):
```

Comment:
maybe `simulate_ideal_trajectories` would be slightly more descriptive in case someone wants to make a different version in the future